.github/workflows/build-publish-dev.yml

```diff
@@ -0,0 +1,17 @@
+name: Publish
+on:
+  pull_request:
+    branches:
+      - develop
```

Comment:
Should this only be published when merge to master instead of develop?

Reply:
This is only to TestPyPi so dev works fine here imo, there will be a similar one for publishing to the actual PyPi which will be for master

Reply:
Sorry missed that it was TestPyPi